src/handlers/webhooks/mod.rs

```diff
@@ -15,10 +16,10 @@ pub struct WebhookConfig {
     events: Vec<WebhookNotificationEvent>,
 }
 
-fn validate_url(url: &str) -> Result<()> {
-    let url = url::Url::parse(url)?;
-    if url.scheme() != "https" {
-        return Err(crate::error::Error::InvalidScheme);
-    }
-    Ok(())
-}
+// fn validate_url(url: &str) -> Result<()> {
```

Comment:
Maybe we can just remove this instead of commenting out?

Reply:
I hastily disabled this, and will be planning to re-enable so I didn't want to loose the context.